sdk/raygun4reactnative.podspec

```diff
@@ -15,14 +15,14 @@ Pod::Spec.new do |s|
   # optional - use expanded license entry instead:
   # s.license    = { :type => "MIT", :file => "LICENSE" }
   s.authors      = { "MindscapeHQ" => "hello@raygun.io" }
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "12.0" }
   s.source       = { :git => "https://github.com/hunteva/raygun4reactnative.git", :branch => "kerwin/refactory/storage" }
 
   s.source_files = "ios/**/*.{h,c,m,swift}"
   s.requires_arc = true
 
-  s.dependency "React"
-  s.dependency "raygun4apple", '~> 1.5.1'
+  s.dependency "React-Core"
+  s.dependency "raygun4apple", '~> 2.0.0'
```

Comment:
Will this affect the customer in a bad way? E.g. backwards compatibility may not be good?

Reply:
As far as I can tell, the only change made in Raygun4Apple version 2.0.0 is to move to a Swift package. That should be all good. However, it will increase the minimum iOS version to 12, which probably necessitates a version bump to 1.3.0!

Reply:
Yes, fair call!